Vonk.Facade.Starter/appsettings.json

```diff
@@ -47,5 +47,22 @@
       "TypeLevelInteractions": "create, search, history, conditional_create",
       "WholeSystemInteractions": "batch, transaction, history, search"
     }
+  },
```

Comment:
I think it is better to disable Authorization by default.
And should the Api source not be enabled at MetadataImportOptions?

Reply:
Discusses with Christiaan:
- yes: disable Authorization (I committed that in this branch)
- Api source should not be enabled in MetadataImportOptions. It will be handled somewhere else